scheduler/src/lib.rs

```diff
@@ -168,6 +168,27 @@ pub mod pallet {
 			Self::deposit_event(Event::CeremonySchedulePushedByOneDay);
 			Ok(().into())
 		}
+
+		#[pallet::weight((1000, DispatchClass::Operational, Pays::No))]
+		pub fn set_phase_duration(
+			origin: OriginFor<T>,
+			ceremony_phase: CeremonyPhaseType,
+			duration: T::Moment,
+		) -> DispatchResultWithPostInfo {
+			T::CeremonyMaster::ensure_origin(origin)?;
+			<PhaseDurations<T>>::insert(ceremony_phase, duration);
+			Ok(().into())
+		}
+
+		#[pallet::weight((1000, DispatchClass::Operational, Pays::No))]
+		pub fn set_next_phase_timestamp(
```

Comment:
We do not have the logic to allow feeless dispatchables yet. Making the feeless now is a potential attack vector, see #101.

Please remove all the `Pays::No`.

Reply:
Also remove them from dispatchables that have had this attribute before.